versions/1.1/SPEC.md

```diff
@@ -0,0 +1,5775 @@
+|-|-|-|
+|`\\`|`\`|`\x5C`||
+|`\n`|newline|`\x0A`||
+|`\t`|tab|`\x09`||
+|`\'`|single quote|`\x22`|within a single-quoted string|
+|`\"`|double quote|`\x27`|within a double-quoted string|
+
+Strings can also contain the following types of escape sequences:
+
+* An octal escape code starts with `\`, followed by 3 digits of value 0 through 7 inclusive.
+* A hexadecimal escape code starts with `\x`, followed by 2 hexadecimal digits `0-9a-fA-F`. 
+* A unicode code point starts with `\u` followed by 4 hexadecimal characters or `\U` followed by 8 hexadecimal characters `0-9a-fA-F`.
+
+### Comments
+
+Comments can be used to provide helpful information such as workflow usage, requirements, copyright, etc. A comment is prepended by `#` and can be placed at the start of a line or at the end of any line of WDL code. Any text following the `#` will be completely ignored by the execution engine, with one exception: within the `command` section, *ALL* text will be included in the evaluated script - even lines prepended by `#`.
+
+There is no special syntax for multi-line comments - simply use a `#` at the start of each line.
+
+```wdl
+# Comments are allowed before versions
+
+version 1.0
+
+# This is how you would
+# write a long
+# multiline
+# comment
+
+task test {
+  #This comment will not be included within the command
+  command <<<
+    #This comment WILL be included within the command after it has been parsed
+    echo 'Hello World'
+  >>>
+
+  output {
+    String result = read_string(stdout())
+  }
+    
+  runtime {
+    container: "my_image:latest"
+  }
+}
+
+workflow wf {
+  input {
+    Int number  #This comment comes after a variable declaration
+  }
+
+  #You can have comments anywhere in the workflow
+  call test
+  
+  output { #You can also put comments after braces
+    String result = test.result
+  }
+}
+```
+
+### Keywords
+
+The following language keywords are reserved and cannot be used to name declarations, calls, tasks, workflows, import namespaces, or struct types & aliases.
+
+```
+Array Boolean Float Int Map None Object Pair String
+
+alias as call command else false if in import input 
+left meta object output parameter_meta right runtime 
+scatter struct task then true workflow
+```
+
+The following keywords should also be considered as reserved - they are not used in the current version of the specification, but they will be used in a future version:
+
+* `Directory`
+* `hints`
+
+### Types
+
+A [declaration](#declarations) is a name that the user reserves in a given [scope](#appendix-b-wdl-namespaces-and-scopes) to hold a value of a certain type. In WDL *all* declarations (including inputs and outputs) must be typed. This means that the information about the type of data that may be held by each declarations must be specified explicitly.
+
+In WDL *all* types represent immutable values.
+
+- Even a `File` represent a logical "snapshot" of the file at the time when the value was created.
```

Comment:
Typo?  this isn't a sentence in my mind:  Even a `File` represent a logical "snapshot" of the file at the time when the value was created.` >>Perhaps change to: >> A `File` represent**s** a logical "snapshot" of the file at the time when the value was created.

Reply:
Also, why are these two sentences bulleted and not just part of the paragraphs above? Are they for some reason different?

Reply:
good suggestion